### PR TITLE
Add MP 7.0 Blog

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

File: posts/2024-12-22-microprofile-7.adoc [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+* `RequestTiming`
+* `ConnectionPool`
+
+To collect and export metrics, enable OpenTelemetry by using the following system property or environment variable:
+
+* System property: `otel.sdk.disabled=false`
+* Environment variable: `OTEL_SDK_DISABLED=false`
+
+You can set configuration properties in any of the link:{url-prefix}/docs/latest/external-configuration.html#default[config sources that are available to MicroProfile Config].
+
```
Comment (Yas-Franco) [inline posts/2024-12-22-microprofile-7.adoc:84]: [#manualMetrics]
=== Use the OpenTelemetry API to mannually create metrics

You can use the OpenTelemetry metrics API to define custom metrics in your application code.

Before instrumenting your code, the following prerequisite is required:

* Third-party APIs must be made visible for your application in the 
link:https://github.com/yasmin-aumeeruddy/mpTelemetry-Demo/blob/main/system/src/main/liberty/config/server.xml#L11-L14[server.xml]:

[source, xml]
----
<webApplication location="demo-microprofile-telemetry-inventory.war" contextRoot="/">
    
    <classloader apiTypeVisibility="+third-party"/>
</webApplication>
----

You can then use the OpenTelemetry API to create custom metrics. 

[source, xml]
```
class WithCounter {
    @Inject
    Meter meter;

    private LongCounter counter;

    @PostConstruct
    public void init() {
        counter = meter
                    .counterBuilder("new_subscriptions")
                    .setDescription("Number of new subscriptions")
                    .setUnit("1")
                    .build();
    }

    void subscribe(String plan) {
        counter.add(1,
            Attributes.of(AttributeKey.stringKey("plan"), plan));
    }
}
```

In this example, `Meter` is used to define an instrument, in this case a Counter. Application code then can record measurement values along with other attributes. Measurement aggregations are computed separately for each unique combination of attributes.

For a full list of available metrics, see the link:https://opentelemetry.io/docs/specs/otel/metrics/api/#meter-operations#default[meter operations] in the OpenTelemetry documentation.

**Review by jim-krueger — APPROVED** [review]
Looks fine from a MP Rest Client perspective

File: posts/2024-12-22-microprofile-7.adoc (hunk 1 above)
Comment (Emily-Jiang) [reply posts/2024-12-22-microprofile-7.adoc:84]: done.

**Review by benjamin-confino — APPROVED** [review]

**Comment by dmuelle** [comment]
Hi @Emily-Jiang - I'm going to merge this so I can review it on the draft site- when you have chance, would you open a PR from your fork to `staging`? I'll add ID review comments in the staging PR. Thanks!

**Pull request merged**